cmd/main.go

```diff
@@ -362,3 +368,12 @@ func initBackupProvider() error {
 	logger.Infow("initialized backup provider", "type", bpString)
 	return nil
 }
+
+func getCompressionMethod() compress.Method {
+	m := viper.GetString(compressionMethod)
+	method, err := compress.MethodFrom(m)
+	if err != nil {
+		logger.Warnw("compression-method unknown", "method", m, "error", err)
```

Comment:
I think this should return an error and fail the application.

Reply:
OK

Reply:
done